std/range/package.d

```diff
@@ -9360,3 +9368,293 @@ if (is(typeof(fun) == void) || isSomeFunction!fun)
 
     auto r = [1, 2, 3, 4].tee!func1.tee!func2;
 }
+
+/**
+Extends the length of the input range `r` by padding out the start of the
```

Comment:
I just want to point out that this sentence also begins with a verb and omits the subject...